### ISSUE TITLE
Feature request: forecast during the day

### ISSUE THREAD

**Issue by sandervandegeijn** [josecastroleon/pimatic-openweather#32]
Currently you can use the forecast for N days ahead, but it would be usefull to have the forecast during the day (5 mins later or N hours later). One usecase is that it would be handy to raise the sun screen when it's going to rain in 5 mins.

**Comment by mwittig** [comment]
Agree, this would be nice. As part of opeweather it is possible to get 3 hourly forecasts if you're lucky as the data isn't provided for all regions. They also were some issues with the forecast5 API which provides the 3-hourly weather forecasts. The forecast16 API we're currently using provides daily forecasts, only. It'd be possible, however, to add another device class for 3 hourly forecasts using the forecast5  as a experimental feature.

Seealso  http://openweathermap.org/api

**Comment by sandervandegeijn** [comment]
It would be a nice addition if it's possible. I would help but I'm not familiar with Coffeescript and Node :)

**Comment by mwittig** [comment]
One comment regarding your using case. In my opinion the data quality provided with the openweather services outside the US/Canada is not good enough for such microweather applications.  On the specific use case it is perhaps more reliable to use a rain sensor (or light sensor so that you raise the sun screen when it is cloudy).

**Comment by sandervandegeijn** [comment]
I noticed, it's okayish, wind speed is a bit problematic as well as it is flapping the status a lot sometimes. Buienradar also has an API that's public, wouldn't that be more useful for the Netherlands?

**Comment by mwittig** [comment]
fyi: The `day` property default value of the `OpenWeatherForecastDevice` has been changed to `2` to obtain the forecast for the next day by default. The property denotes the forecast day starting from 1 to 16 where 1 means today. In other words, if you set `day` to `1` you'll get the forecast for the same day.
